index.bs

```diff
@@ -94,13 +104,17 @@ ISSUE: Should this specification also define a structural brand stricter than 'm
 
 <h3 id="image-item">AV1 Image Item</h3>
 
-<p>When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>, it is called an <dfn>AV1 Image Item</dfn>, and SHALL obey the following constraints:
-<ul>
-<li>The item data SHALL be the content of a single valid [=AV1 Sync Sample=] as defined in [[!AV1-ISOBMFF]] with the additional constraints that there SHALL be only one [=Sequence Header OBU=] and it SHALL have its <code>[=still_picture=]</code> flag set to 1 and SHOULD have its <code>[=reduced_still_picture_header=]</code> flag set to 1.
-<li>The item SHALL be associated with an [=AV1 Item Configuration Property=].</li>
-</ul>
+When an image item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>, it is called an <dfn>AV1 Image Item</dfn>, and shall obey the following constraints:
+    
+  - The [=AV1 Image Item=] is a coded image item in which the coded image is a single valid [=AV1 Sample=] as defined in [[!AV1-ISOBMFF]].
+  - The [=AV1 Image Item=] shall be associated with an [=AV1 Item Configuration Property=].
 
-</p>
+The [=AV1 Sample=] that is the coded image of the [=AV1 Image Item=], is called <dfn>AV1 Image Item Data</dfn>. 
```

Comment:
The term "Sample" is confusing when used this way. Please replace with:
"The content of an [=AV1 Image Item=] is called the <dfn>AV1 Image Item Data</dfn>. "

Reply:
Done